Formula/y/yewtube.rb

```diff
@@ -130,8 +130,11 @@ def caveats
   end
 
   test do
-    Open3.popen3("#{bin}/yt", "/Drop4Drop x Ed Sheeran,", "d 1,", "q") do |_, _, stderr|
-      assert_empty stderr.read, "Some warnings were raised"
-    end
+    system bin/"yt",
+      "set checkupdate false,",
+      "set ddir \"#{testpath}\",",
+      "/youtube-dl test video,", "d 1,", "q"
+    downloaded_file = Dir.glob(testpath/"mps/*").first
+    assert_match(/^(WebM)|(.*MP4.*)$/, shell_output("file --brief \"#{downloaded_file}\""))
```

Comment:
```suggestion
    assert_match(/^(WebM)|(.*MP4.*)$/, shell_output("file --brief '#{downloaded_file}'"))
```

Reply:
Thanks! This surfaced the fact that the title of the video (probably intentionally for CI sanity-checking) has a bunch of odd characters in it, including quotes. So I just got rid of the `shell_output()` entirely and switched to `Utils.safe_popen_read()`.